requirements.txt

```diff
@@ -9,6 +9,7 @@ dnspython==2.2.1
 email-validator==1.3.0
 Flask==2.2.2
 Flask-Admin==1.6.0
+Flask-APScheduler

```

Comment:
Are there version constraints?

We probably should trim the list to only the direct dependencies and be more general about versions...

Reply:
Did a little digging and didn't find any obvious documentation indicating what versions of flask are supported, etc. I'll pin this at the current version for now, but I agree, we should probably loosen up these requirements at some point.